addons/l10n_dz/models/invoice.py

```diff
@@ -0,0 +1,9 @@
+from odoo import api, models
```

Comment:
```suggestion
# -*- coding: utf-8 -*-

from odoo import api, models
```

Reply:
Could you please rename the file for `account_move.py`

Reply:
My bad, it looks like it's no more necessary to add `# -*- coding: utf-8 -*-`
You can remove it. Sorry :sweat_smile: